api/api_types.go

```diff
@@ -158,6 +158,23 @@ type TransactionReference struct {
 	Height uint32 `json:"blockHeight"`
 	Index  uint32 `json:"transactionIndex"`
 }
+type TransactionsCost struct {
+	Costs transactionCosts `json:"costs,omitempty"`
+}
+type transactionCosts struct {
```

Comment:
Is this definition actually needed? Cannot we manage it with a `map[string]uint64` api/chain.go does?

Reply:
loved this solution. made everything so simpler, thanks a million